src/logger/logger.service.ts

```
@@ -1,38 +1,124 @@
+    const transports: winston.transport[] = [
+      new winston.transports.Console({
+        format: winston.format.combine(
+          winston.format.colorize(),
+          winston.format.simple()
+        )
+      }),
+      new winston.transports.File({ filename: 'error.log', level: 'error' }),
+      new winston.transports.File({ filename: 'combined.log' }),
+    ];
+
     this.logger = winston.createLogger({
```

ESLint

[error] Replace `····` with `↹↹`

(prettier/prettier)